monailabel/monailabel_HelloWorld_radiology_3dslicer.ipynb

```diff
@@ -102,7 +102,9 @@
    "source": [
     "### 1.4 Starting MONAI Label Server\n",
     "\n",
-    "The segmentation_spleen model will automatically be downloaded when starting the MONAI Label server. Network and checkpoint will be processed within the initialization steps of monailabel command."
+    "The segmentation_spleen model will automatically be downloaded when starting the MONAI Label server. Network and checkpoint will be processed within the initialization steps of monailabel command.\n",
+    "\n",
```

Comment:
"The `segmentation_spleen` model will be automatically downloaded upon starting the MONAI Label server. The network and checkpoint will be processed during the initialization steps of the `monailabel` command.